packages/card-picker/addon/templates/components/card-picker-toolbox.hbs

```diff
@@ -1,10 +1,20 @@
-<div class="heading">
+<div class="heading" data-card-picker-toolbox-header>
   {{cardstack-query-editor query=query update=(action (mut query))}}
+  <button
+    onclick={{action "closePicker"}}
+    class="cardstack-card-picker--close-button"
+    data-test-card-picker-close-button
+  >
+    Close
+  </button>
 </div>
 
 <div class="cards-list">
   {{#cardstack-search query=query as |content|}}
-    <div role="button" onclick={{action "select" content}}>
+    <div
+      role="button"
+      onclick={{action "select" content}}
+    >
```

Comment:
unrelated change?

Reply:
how did it pass before this PR?

Reply:
I actually don't know, I guess it was before template linting was enabled?